src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -527,7 +530,21 @@ def build(self, notification_uuid: str | None = None) -> SlackBlock:
         # build title block
         title_text = f"<{title_link}|*{escape_slack_text(title)}*>"
 
-        if self.group.issue_category == GroupCategory.ERROR:
+        # build up actions text
```

Comment:
I think if we break up this function a little more and consolidate some things, it will make this look less messy and easier to read and add future changes. Let's try it and brainstorm in another PR